lib/nrf_modem_lib/Kconfig

```diff
@@ -6,10 +6,12 @@
 menuconfig NRF_MODEM_LIB
 	bool "Modem library"
 	depends on (SOC_SERIES_NRF91X && TRUSTED_EXECUTION_NONSECURE) || SOC_NRF9280_CPUAPP
-	select EXPERIMENTAL if SOC_NRF9280_CPUAPP
 	select NRF_MODEM
+	select EXPERIMENTAL if SOC_NRF9280_CPUAPP
 	imply NET_SOCKETS_OFFLOAD
 	imply NET_SOCKETS_POSIX_NAMES if !POSIX_API
+	imply NET_IPV4
+	imply NET_IPV6
```

Comment:
`NET_IPV*` needs `NETWORKING` to be enabled to select, `NET_SOCKETS_OFFLOAD` needs `NET_SOCKETS` to be enabled to select, also `NET_SOCKETS_POSIX_NAMES` has been deprecated for a very long time and is about to vanish in the upmerge, the migration guide states:
```
* The :kconfig:option:`CONFIG_NET_SOCKETS_POSIX_NAMES` option has been
  removed.  It was a legacy option and was used to allow user to call BSD
  socket API while not enabling POSIX API.  This removal means that in order
  to use POSIX API socket calls, one needs to enable the
  :kconfig:option:`CONFIG_POSIX_API` option.  If the application does not want
  or is not able to enable that option, then the socket API calls need to be
  prefixed by a ``zsock_`` string.
```

Reply:
seems OK for imply actually